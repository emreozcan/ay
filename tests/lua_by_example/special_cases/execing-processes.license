https://luabyexample.techplexlabs.com/execing-processes/
Lua by Example: Exec'ing Processes
by Blake Bourque

This work is licensed under a Creative Commons Attribution 3.0 Unported License.