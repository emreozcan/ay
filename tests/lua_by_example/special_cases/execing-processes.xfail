We do not have io.popen()
