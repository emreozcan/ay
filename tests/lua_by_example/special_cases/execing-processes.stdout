successfully created directory
output is:	Tue Jan 19 03:14:07 UTC 2038
